liquidprompt

```diff
@@ -584,6 +584,7 @@ fi
 # Where are we? #
 #################
 
+if ! type _lp_connection &> /dev/null; then
```

Comment:
This breaks my usual developement workflow: this check stops from updating liquidprompt's code by just reloading it in the same shell.
We should find a better way.